src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,40 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
 
     public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
let's split this method into three small ones:
1. read from file
2. calculate statistic
3. write statistic to file